src/palo-alto-networks/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+1.1.0
```

Comment:
The version is not correct as 1.0.0 has not been released yet. It's better to use `1.0.0b2`